tests/Backend/Snowflake/TimestampTest.php

```diff
@@ -10,60 +10,35 @@
 
 class TimestampTest extends WorkspacesTestCase
 {
+    const TIMESTAMP_FORMAT = '/\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}/';
```

Comment:
Měl by to být end-to-end regex - tohle matchuje třeba i `lorem ipsum 1234-12-12 11:22:33 some 💯 more shit` Což asi nechceme... takže `/^...$/`